gen-ai/Assistants/travel_planner/README.md

```diff
@@ -0,0 +1,122 @@
+# Travel Planner Assistant
+
+## Overview
+This sample provides a guide to use the new Web Browse tool with the Azure OpenAI Assistants. 
+This tool is based on Bing Search API and allows to easily implement a public web data grounding.
+
+
+Given LLMs have data available only up to a cut off date, it might not handle questions that require up-to-date information.
+And this is where the Web Browse tool comes in handy!
+
+
+## Objective
+The objective of this sample is to create an OpenAI assistant that can help you plan your trip and which will use the Web Browse 
+tool whenever it needs to get the latest information related to the trip.
+
+The assistant will be implemented through a CLI in python (command line interface) which the user can use to interact with the assistant.
+
+By the end of this tutorial, you should be able to:
+- Create an OpenAI assistant that uses the Web Browse tool
+- Understand how to instruct the assistant to use the Web Browse tool
+
+## Programming Languages
+- Python
+
+## Estimated Runtime: 10 mins
+
+## Pre-requisites
+- A Azure OpenAI resource (API key + endpoint)
```

Comment:
It might be interesting to point here to some web search tool documentation, but Im not sure we already have any

Reply:
I'll take a look and try to run this in a little bit. I split this repo off into its own repository. Do we want to try and merge the code there and polish assistants and that repo?

Reply:
thanks @codebytes !

regarding the split, I was a bit on the fence. In a way I dont think this is an accelerator, but just a small sample to show how to use the feature (like a jupyter notebook).

Is this a use case that we discusses previously about having its own repo?